src/application/mLoginBox.cpp

```diff
@@ -3,16 +3,25 @@
  * All rights reserved. Distributed under the terms of the MIT license.
  */
 #include <Catalog.h>
+#include <GraphicsDefs.h>
+#include <InterfaceDefs.h>
 #include <LayoutBuilder.h>
+#include <stdlib.h>
 #include "mLoginBox.h"
+#include "View.h"
+#include "Box.h"
```

Comment:
If you are calling the Haiku API headers, they should be called as `#include <header.h>`.